FurCoNZ/FurCoNZ.csproj

```diff
@@ -25,5 +25,8 @@
   <ItemGroup>
     <Folder Include="ViewModels\" />
     <Folder Include="Views\Order\" />
+    <Folder Include="Views\Order\EditorTemplates\" />
+    <Folder Include="Helpers\" />
+    <Folder Include="Views\Order\DisplayTemplates\" />

```

Comment:
I would avoid adding empty folders into the `.csproj`. they only serve to add clutter IMO.

Reply:
If they're not empty. then removing these entries is "safe" and will not break any code.

Reply:
Resolved in a5821ad